e2e/scenario_test.go

```diff
@@ -1198,3 +1198,37 @@ func Test_AzureLinuxV2_KubeletCustomConfig(t *testing.T) {
 		},
 	})
 }
+
+func Test_Ubuntu2204ARM64_KubeletCustomConfig(t *testing.T) {
+	RunScenario(t, &Scenario{
+		Tags: Tags{
+			KubeletCustomConfig: true,
+		},
+		Description: "tests that a node on ubuntu 2204 ARM64 bootstrapped with kubelet custom config",
+		Config: Config{
+			Cluster: ClusterKubenet,
+			VHD:     config.VHDUbuntu2204Gen2Arm64Containerd,
+			BootstrapConfigMutator: func(nbc *datamodel.NodeBootstrappingConfiguration) {
+				nbc.IsARM64 = true
```

Comment:
You wouldn't need to explicitly state it's ARM64 if you inferred it from the VHD arch in scenario_helpers_test.go prepareAKSNode func. But maybe it's better to have it explicitly stated in the test.

```
func prepareAKSNode(ctx context.Context, s *Scenario) {
...
nbc := getBaseNBC(s.T, s.Runtime.Cluster, s.VHD)
if(s.VHD.Arch == "arm64") {
    nbc.IsARM64 = true
}
```